packages/@aws-cdk/aws-appsync/test/integ.graphql.ts

```diff
@@ -17,6 +35,7 @@ const customerTable = new Table(stack, 'CustomerTable', {
         name: 'id',
         type: AttributeType.STRING,
     },
+    removalPolicy: RemovalPolicy.DESTROY,
```

Comment:
Was the retain removal policy causing failures in the integ tests for you?

Reply:
No, but it was annoying me to have a ton of left over tables in my account.

Reply:
Agreed. Lets pull it out of this PR so we can double check with other team members on implications before doing this. Most likely fine though for a future chore.

Reply:
Done. I opened https://github.com/aws/aws-cdk/pull/6401.